ledger/sandbox-common/src/main/scala/platform/sandbox/config/SandboxConfig.scala

```diff
@@ -64,14 +64,20 @@ final case class SandboxConfig(
     sqlStartMode: Option[PostgresStartupMode],
     enableAppendOnlySchema: Boolean,
     enableCompression: Boolean,
+    maxDeduplicationDuration: Option[Duration],
```

Comment:
Same consideration here as for `kvutils.app.Config`.

Reply:
Will do the changes in another PR